package.json

```diff
@@ -15,9 +15,10 @@
     "build:tsc": "tsc -p test/",
     "build:webpack": "webpack --config webpack.config.js",
     "check_version": "echo ${npm_package_version} $(npm view ${npm_package_name} version) | awk '{ if ($1 == $2) { exit 1 }}'",
-    "start": "npm-run-all -p watch:watch-tsc watch:watch-quicktests webpack-dev-server",
-    "start:watch-quicktests": "grunt watch-quicktests",
-    "start:watch-tsc": "tsc -w -p test/",
+    "dev:server": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js", 
+    "start": "npm-run-all -p watch:watch-tsc watch:watch-quicktests dev:server",
```

Comment:
yeah sorry i screwed this up; fixed it in https://github.com/palantir/plottable/pull/3225 , that version look good?